### ISSUE TITLE
Ensure that licenses are followed for all images used

### ISSUE THREAD

**Issue by alexgleason** [Uptwinkles/uptwinkles#17]
I know that for at least the Libreboot photo we need to specify that it's licensed under GPL somewhere, either in the readme or on the site. We could also just take a new pic ourselves of one of our computers.

Anything from librestock is in public domain, so no issue there. I'm unsure where the Tux came from.

**Comment by marykatefain** [comment]
Yeah this sortof goes hand in hand with #5, most of these were never
intended to stay
